cmd/root.go

```diff
@@ -55,7 +56,13 @@ There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 			if err != nil {
 				return err
 			}
-			policiesFetcher, err := policies.NewFetcher(kubewardenNamespace)
+			skippedNsCSV, err := cmd.Flags().GetString("skipped-ns")
+			if err != nil {
+				return err
+			}
+			skippedNs := strings.Split(skippedNsCSV, ",")
```

Comment:
I think you can use [`StringSliceVarP`](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.StringSliceP) instead of `StringP`. It splits the values by comma for you.

Furthermore, it makes more easy to build the yaml in the Helm chart templates. Because we do not need to concat all the skipped namespaces. We can just pass every skipped namespace in a separated flag.

Reply:
Indeed, better! Done in https://github.com/kubewarden/audit-scanner/pull/53/commits/18d37612e41788eeadeb3bd78ed0c0be85a16b4c.